crates/sgx_validation/src/lib.rs

```diff
@@ -75,42 +83,68 @@ impl Sgx {
         //
         // Additionally, we do this check early to be defensive.
         if cri.public_key.algorithm != pck.subject_public_key_info.algorithm {
-            return Err(anyhow!("sgx pck algorithm mismatch"));
+            bail!("sgx pck algorithm mismatch");
```

Comment:
If you're changing this, then use `ensure!` (or drop the diff and file a debt issue to refactor)

Reply:
Fixed